docs/pages/installation.mdx

```diff
@@ -107,21 +107,197 @@ information on obtaining Teleport binaries compatible with Teleport Cloud.
+
+| Image name | Teleport version | Includes troubleshooting tools | Image base |
+| - | - | - | - |
+| `(=teleport.latest_ent_docker_image=)` | No | [Distroless Debian 11](https://github.com/GoogleContainerTools/distroless) |
+| `(=teleport.latest_ent_debug_docker_image=)` | Yes | [Distroless Debian 11](https://github.com/GoogleContainerTools/distroless) |
+
+We also provide the following images for FIPS builds of Teleport Enterprise:
+
+| Image name | Includes troubleshooting tools | Image base |
+| - | - | - |
+| `public.ecr.aws/gravitational/teleport-ent:(=teleport.major_version=)-fips` | Yes | [Ubuntu 20.04](https://hub.docker.com/\_/ubuntu) |
+| `(=teleport.latest_ent_docker_image=)-fips` | Yes | [Ubuntu 20.04](https://hub.docker.com/\_/ubuntu) |
 
-We provide pre-built `amd64`, `arm`, and `arm64` Docker images for every version of Teleport Enterprise.
+For testing, we always recommend that you use the latest release version of
+Teleport Enterprise, which is currently `(=teleport.latest_ent_docker_image=)`.
 
-(!docs/pages/includes/enterprise/docker-images.mdx!)
+[Ubuntu 20.04](https://hub.docker.com/\_/ubuntu)-based images for non-FIPS 
+Teleport are available from our [Legacy Amazon ECR Public repository](https://gallery.ecr.aws/gravitational/teleport-ent).
 
 </TabItem>
 </Tabs>
 
+### Running Teleport on Docker
+
+When running a container from one of the images listed above, consider the
+container equivalent to running the `teleport` binary. The Teleport container
+requires access to a file system and network ports.
+
+Our [Helm charts](#helm) take care of the Teleport container's system
+requirements for you if deploying Teleport on Kubernetes, so these instructions
+are relevant only when running a standalone Teleport container in some other
+environment.
+
+#### Configuration
+
+Teleport processes read their configuration from a local file path, which is
+`/etc/teleport.yaml` by default. Make sure this file path is mounted to your
+Teleport container.
+
+#### Data directory
+
+All Teleport processes read from and write to a data directory, which by default
+is `/var/lib/teleport`. Make sure the data directory is mounted to your Teleport
+container.
+
+#### License file
+
+If your Teleport Enterprise container runs the Auth Service, you will need to
+give it access to a license file at the path named in the configuration, which
+is `/var/lib/teleport/license.pem` by default. Make sure a license exists at
+this location in the Teleport container's data directory.
+
+#### Other file paths
+
+Depending on the configuration settings you assign on your Teleport container,
+you will need to make sure that any file paths you name are mounted on the
+container. 
+
+For example, if you are running the Teleport Proxy Service on a container, you
+need to mount this directory to your Teleport container, then assign the
+following fields in the container's configuration file to the appropriate paths:
```

Comment:
I suggest rephrasing this paragraph. It looks like it was pulled from another section where "this directory" was already explained, but lacks context here. We should explain that the example is specifically for mapping keypairs from a directory in the container host. Maybe something like:

```suggestion
For example, if you are running the Teleport Proxy Service on a container and
are providing externally generated HTTPS keypairs, you need to mount the directory
containing those keys to your Teleport container, then assign the
following fields in the container's configuration file to the appropriate paths:
```